content/en/docs/Core concepts/concurrency.md

```diff
@@ -0,0 +1,59 @@
+---
+title: "Tuning concurrency"
+date: 2020-11-02
+weight: 3
+---
+
+Baker filters and ouput can be configured with a high concurrency profile (input doesn't support
+parallel processing).
+
+This means that those components will process Records in parallel during the pipeline,
+improving performances but adding a small cons: Records order isn't guaranteed anymore
+(due to possible different speed processing between concurrent components).
+
+{{% alert title="Default behaviour" color="primary" %}}
+The default behaviour of Baker is to use concurrency for both filters and output.
+{{% /alert %}}
+
+### Concurrent filtering
+
+The filters concurrency can be set defining the `procs` key in the `[filterchain]` section:
+
+```toml
+[filterchain]
+procs=16
+```
+
+The default value is **16**.  
+To disable concurrency, set **procs=1**.
+
+### Concurrent output
+
+The output concurrency can be set defining the `procs` key in the `[output]` section:
+
+```toml
+[output]
+procs=32
+```
+
+The default value is **32**.  
+To disable concurrency, set **procs=1**.
+
+#### Output concurrency support
+
+At the moment it is not possible to know whether an output supports concurrency. A good guess is to
```

Comment:
Uhm not sure I like that we don't know that and we have to guess :-)
Outputs should definitely document if they support concurrency or not. Speaking about sharding here adds a bit of confusion IMHO.